programs/server/config.xml

```diff
@@ -1297,6 +1297,30 @@
         <ttl>event_date + INTERVAL 30 DAY</ttl>
     </blob_storage_log>
 
+    <!-- Configure system.dashboards for dashboard.html.
+
+         Should have the following parameters:
+         - seconds
+         - rounding
```

Comment:
It is good to mention they can have different custom parameters as well with these two

Reply:
Updated